src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue

```diff
@@ -64,35 +70,43 @@
 import { computed, reactive } from 'vue';
 
 import {
-    PPaneLayout, PPanelTop, PBoard, PLabel,
+    PPaneLayout, PPanelTop, PBoard, PLabel, PTextPagination,
 } from '@spaceone/design-system';
-import type { BoardSet } from '@spaceone/design-system/types/data-display/board/type';
 
 import type { DashboardConfig } from '@/services/dashboards/config';
 import { DASHBOARD_TEMPLATES } from '@/services/dashboards/default-dashboard/template-list';
 
-const emit = defineEmits(['set-template']);
 
+const emit = defineEmits(['set-template']);
 
 type dashboardTemplateBoardSet = DashboardConfig & { leftIcon: string, iconButtonSets: Array<any> };
 
 const state = reactive({
     selectedTemplate: {} as DashboardConfig,
+    // default templates
+    defaultTemplateThisPage: 1,
     defaultBoardSets: computed<dashboardTemplateBoardSet[]>(() => Object.values(DASHBOARD_TEMPLATES).map((d: DashboardConfig) => ({
         ...d,
         // below values are used only for render
         leftIcon: d.description?.preview_image ?? '',
         iconButtonSets: [{ iconName: 'ic_external-link', tooltipText: 'Preview', eventAction: () => {} }],
-    }))),
-    existingBoardSets: computed<BoardSet[]>(() => []),
+    })).splice(10 * (state.defaultTemplateThisPage - 1), 10 * state.defaultTemplateThisPage - 1)),
```

Comment:
suggestion: `splice()` changes origin data, so I think it's safer to use `slice()`!

Reply:
Oh great!